HeatApp/components/screens/StartSettingsScreen.js

```diff
@@ -1,25 +1,31 @@
 import React from "react";
 import {
-  Text, StyleSheet, Dimensions, TouchableOpacity
+  Text, StyleSheet, Dimensions, TouchableOpacity, View
 } from "react-native";
 import colors from './../../config/colors';
+import { ApiService } from './../../models/ApiService';
 
 const {width: WIDTH} = Dimensions.get('window');
```

Comment:
It's not part of your request, but mb we'll fix it there if it's not a problem)
We should move all dimensions to separate file.

Reply:
Done